processus-traitement-donnees.md

```diff
@@ -59,24 +58,29 @@ Le workflow classique d'intégration consiste à:
 
 - Constituer un objet `batch` listant les fichiers de données à importer (cf [procédure avec `prepare-import`](procedure-import-donnees.md)), puis l'insérer dans la collection `Admin`.
 
-- Lancer l'API:
+- Mettre à jour la commande `sfdata`, depuis `ssh centos@labtenant -t tmux att`:
 
   ```sh
-  $ go build && ./dbmongo
+  killall sfdata
+  cd opensignauxfaibles
+  git pull
+  go build # pour compiler la commande ./sfdata
+  ./sfdata --help
   ```
 
 - Appeler séquentiellement les fonctions d'intégration (et de contrôle) pour importer, compacter les données puis calculer les variables avec les options idoines:
 
   ```sh
   # 1. Import
-  $ http :3000/api/data/check batch="1904"
-  $ http :3000/api/data/import batch="1904"
+  ./sfdata check --batch="1904"
+  ./sfdata import --batch="1904"
   # 2. Compactage
-  $ http :3000/api/data/validate collection="ImportedData"
-  $ http :3000/api/data/compact fromBatchKey="1904"
-  # 3. Calcul
-  $ http :3000/api/data/validate collection="RawData"
-  $ http :3000/api/data/reduce batch="1904"
+  ./sfdata validate --collection="ImportedData"
+  ./sfdata compact --fromBatchKey="1904"
```

Comment:
je trouve cette irruption de camelCase un peu étonnante au milieu de ce beau kebabCase :)
```suggestion
  ./sfdata compact --from-batch="1904"
```
ou encore, dans une conception plus temporelle
```suggestion
  ./sfdata compact --since-batch="1904"
```